eslint.config.js

```diff
@@ -17,14 +17,31 @@ export default defineConfig([
   globalIgnores(['dist']),
   {
     extends: [
+      // https://github.com/eslint/eslint/blob/20a9e59438fde3642ab058cc55ee1b9fa02b6391/packages/js/src/configs/eslint-recommended.js
       eslint.configs.recommended,
+
+      // https://github.com/sindresorhus/eslint-plugin-unicorn/tree/main/rules
       unicorn.configs['flat/all'],
+
+      // https://github.com/bryanrsmith/eslint-plugin-sort-class-members/blob/main/src/index.js
       sortClassMembers.configs['flat/recommended'],
+
+      // https://github.com/import-js/eslint-plugin-import/blob/main/config/recommended.js
       importPlugin.flatConfigs.recommended,
+
+      // https://github.com/43081j/eslint-plugin-lit/blob/master/src/configs/recommended.ts
       compat.extends('plugin:lit/recommended'),
+
+      // https://github.com/open-wc/open-wc/blob/c6f14851e1ec257cc03726c058d7fbf7223471ba/packages/eslint-plugin-lit-a11y/lib/index.js
```

Comment:
Same question:

```suggestion
      // https://github.com/open-wc/open-wc/blob/master/packages/eslint-plugin-lit-a11y/lib/index.js
```

(and for below)